.travis.yml

```diff
@@ -1,7 +1,7 @@
 ---
 language: node_js
 node_js:
-  - "8"
+  - "10"
```

Comment:
isn't this a breaking change that would need a major version release?

Reply:
@Turbo87 ember-cli dropped support for Node 8 in a minor version of 3.x (3.16.0).

addons are forced to  follow suit because there isn't much of an option when the ecosystem is broken on Node 8.

Reply:
> ember-cli dropped support for Node 8 in a minor version of 3.x

ember-cli uses a `~` constraint for the dependency and explicitly declared this as it's policy already years ago. regular addons are installed with `^` constraints though...

> addons are forced to follow suit because there isn't much of an option when the ecosystem is broken on Node 8.

I totally disagree. Just because some dependency down the line does not do this doesn't mean that the addon can't bump their major version. it's not a reason to make the situation even worse.

Reply:
![image](https://user-images.githubusercontent.com/3108309/80894624-763bd280-8c91-11ea-8751-2a4e6da4c327.png)

Is this point I'm making

Reply:
again, I totally disagree. the `~` constraint on the ember-cli dependency should protect you from accidentally getting an incompatible ember-cli release that no longer supports your Node version of choice. CI will only break if you explicitly upgrade it to a version that does indeed no longer support that Node version.

Reply:
I had no idea there was such a policy that enables ember-cli to disregard semver.

Reply:
https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md

and since this is a published policy it is not in disregard of semver. the problem only exists if a dependency has not published any policy and is then doing such changes without warning. if this policy is known from the start you at least have the option to use the `~` constraint to prevent issues like this.

Reply:
and also: https://emberjs.com/blog/2016/09/07/ember-node-lts-support.html 😉

Reply:
I've never heard of that in semver, thanks for the info though.

Reply:
yeah, it's unfortunate that `package.json` does not support comments, otherwise this would be easy to document 😅